flow/project.py

```diff
@@ -405,6 +415,18 @@ def __call__(self, job=None):
             return self._cmd.format(job=job)
 
 
+def _create_metacondition(condition_dict, *other_funcs):
```

Comment:
_I had left the most important comment as part of my review on this function, but somehow it got lost during the submission? Anyways, I'll just write it again..._

Here we make an assumption here that all meta-conditions use the `all` function to aggregate multiple condition functions. I think this should be either made explicit in the function name or the aggregation function (`all`, `any`, etc.) should be an argument.

Reply:
This is a good point. For now I'm going to just make it explicit in the name because I'm not exactly certain how the `composed_of` logic for graphs would work for other aggregation functions and I don't see a purpose to generalizing until we have an actual use-case.

Reply:
I think it's fine to move forward with this without much further consideration since this is purely internal anyways.